package.json

```diff
@@ -14,5 +14,10 @@
   "bugs": {
     "url": "https://github.com/hashgraph/hedera-smart-contracts/issues"
   },
-  "homepage": "https://github.com/hashgraph/hedera-smart-contracts#readme"
+  "homepage": "https://github.com/hashgraph/hedera-smart-contracts#readme",
+  "devDependencies": {
+    "@hashgraph/hardhat-hethers": "^1.0.2",
```

Comment:
Q: Why do we need `hardhat-hethers` vs just hardhat?
If this is only needed to workaround the contract address after deployment then it's fine to have it but we should import hardhat explicitly and only use `@hashgraph/hethers` for the single purpose.
For this repo let's keep it to the bare minimum of local node and hardhat with needed configurations to work